### PR TITLE
Paragraph line height

### PR DESCRIPTION
**Pull request by johnhwhite**
Create `.sky-paragraph` and `.sky-paragraph-wrapper` for setting line height on paragraphs in default theme.

### CODE REVIEW

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-theme?_sp.spa=pr-sky-paragraph](https://developer.blackbaud.com/skyux-theme?_sp.spa=pr-sky-paragraph)

File: src/app/visual/theming/theming-demo.component.html [hunk 1]
```diff
@@ -85,6 +86,21 @@ <h2 class="sky-font-heading-5">Heading 5 (by CSS class)</h2>
     <div class="padding-test sky-padding-horizontal-xl">Padding horizontal extra-large</div>
   </ng-container>
 
+  <ng-container *ngIf="section === 'paragraph'">
+    <div class="sky-paragraph-wrapper">
```
Comment (Blackbaud-PaulCrowder) [inline src/app/visual/theming/theming-demo.component.html:90]: Since you removed the wrapper CSS class, you can remove this element as well.

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-theme?_sp.spa=pr-sky-paragraph](https://developer.blackbaud.com/skyux-theme?_sp.spa=pr-sky-paragraph)

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-theme?_sp.spa=pr-sky-paragraph](https://developer.blackbaud.com/skyux-theme?_sp.spa=pr-sky-paragraph)

**Review by Blackbaud-ToddRoberts — APPROVED** [review]

**Pull request merged**